app/Livewire/Server/Show.php

```
@@ -67,13 +67,21 @@ class Show extends Component
 
     public string $serverTimezone;
 
+    public ?string $hetznerServerStatus = null;
+
+    public bool $hetznerServerManuallyStarted = false;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $hetznerServerManuallyStarted. Keep variable name length under 20. (undefined)

(LongVariable)